src/components/PersonLink.tsx

```diff
@@ -0,0 +1,67 @@
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+
+import { Person } from '../types';
+
+interface Props {
+  person: Person;
+  activeSlug: string | undefined;
+  search: string;
+}
+
+export const PersonLink: React.FC<Props> = ({
+  person: {
+    slug,
+    sex,
+    name,
+    born,
+    died,
+    motherName,
+    fatherName,
+    mother,
+    father,
+  },
+  activeSlug,
+  search,
+}) => {
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({ 'has-background-warning': activeSlug === slug })}
+    >
```

Comment:
It's a good practice to move complex expressions to variables to improve readability. The expression `classNames({ 'has-background-warning': activeSlug === slug })` can be moved to a variable before the return statement.